src/main/java/com/wavefront/opentracing/WavefrontTracer.java

```diff
@@ -459,6 +475,22 @@ public Builder excludeJvmMetrics() {
       return this;
     }
 
+    /**
+     * Set custom RED metrics tags. If the span has any of the tags, then those get reported to
+     * the span generated RED metrics.
+     * Example - If you have a span tag of 'tenant-id', that you also want to be propagated to the
+     * RED metrics then you would call this method and pass in 'tenant-id' to the set.
+     * Caveat - ensure that redMetricsCustomTags are low cardinality tags.
+     *
+     * @param redMetricsCustomTags set of custom tags you want to report for the span-generated RED
+     *                             metrics.
+     * @return {@code this}
+     */
+    Builder redMetricsCustomTags(Set<String> redMetricsCustomTags) {
```

Comment:
nit: can we call the method and variable as`redMetricsCustomTagKeys` since they are keys which should be scanned for?